pages/common/ical.md

```diff
@@ -0,0 +1,15 @@
+# ical
+
+> Program to display english Calendar on your terminal
+
```

Comment:
```suggestion
More information: <https://manned.org/ical>.

```
We try to put links to man pages or home pages into every page. If you find any official link for `ical` feel free to replace my suggested link.

Reply:
@marchersimon you frequently by mistake omit the `>` at the beginning ;)

Reply:
Oh, thanks for reminding me